crates/logging/logger/Cargo.toml

```
@@ -16,4 +16,7 @@ cgp                                 = { workspace = true }
 hermes-relayer-components           = { workspace = true }
 hermes-relayer-components-extra     = { workspace = true }
 hermes-logging-components           = { workspace = true }
-hermes-tracing-logging-components   = { workspace = true }
\ No newline at end of file
+hermes-tracing-logging-components   = { workspace = true }
+
+tracing                             = { workspace = true }
+tracing-subscriber                  = { workspace = true }
```

New line at the eof.

```suggestion
tracing-subscriber                  = { workspace = true }

```